### PR TITLE
Move default goth assets to yagna and clean-up provider assets

### PR DESCRIPTION
**Pull request by evik42**
Resolves: #2634 #2613

### CODE REVIEW

File: goth_tests/assets/docker/yagna-goth-deb.Dockerfile
Comment (MrDarthShoe) [inline goth_tests/assets/docker/yagna-goth-deb.Dockerfile:1]: We have `yagna-goth.Dockerfile` and `yagna-goth-deb.Dockerfile`. Where `deb` one is used?

File: goth_tests/assets/docker/README.md
[the remark this answers is not in the document]
Comment (evik42) [reply goth_tests/assets/docker/README.md:1]: These are the assets that are used by all tests in the top folder. I tried to do test specific ones previously and for some reason the docker files were still read from this directory (this is where previously the default assets were created by goth).
So without spending too much time in goth to figure out why the configuration parameter is ignored, this is the place to put them.

File: goth_tests/assets/docker/yagna-goth-deb.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# debian:bullseye-slim, 2021-05-12
+FROM debian@sha256:e827c9bc6913625ec75c8b466e6e79a6b936d0801956be607bce08a07078f57a
```
[the remark this answers is not in the document]
Comment (evik42) [reply goth_tests/assets/docker/yagna-goth-deb.Dockerfile:2]: I copied the default assets, which were copied into this folder at the start of each goth build from goth. This is a first step and we can improve from here gradually.

File: goth_tests/assets/web-root/params.json
[the remark this answers is not in the document]
Comment (evik42) [reply goth_tests/assets/web-root/params.json:1]: I copied the default assets, which were copied into this folder at the start of each goth build from goth. This is a first step and we can improve from here gradually.

File: goth_tests/assets/docker/yagna-goth-deb.Dockerfile
Comment (evik42) [reply goth_tests/assets/docker/yagna-goth-deb.Dockerfile:1]: I copied the default assets, which were copied into this folder at the start of each goth build from goth. This is a first step and we can improve from here gradually.

**Pull request merged**